build/vsts_build.yaml

```diff
@@ -183,6 +183,19 @@ jobs:
       optimizationInputsDropNamePrefix: OptimizationInputs/$(System.TeamProject)/$(Build.Repository.Name)
     condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
 
+  - task: NuGetCommand@2
+    displayName: "Install dotnet-format"
+    inputs:
+      command: 'custom'
+      arguments: 'install dotnet-format -ConfigFile $(System.DefaultWorkingDirectory)\NuGet.config -OutputDirectory $(System.DefaultWorkingDirectory)\tools'
```

Comment:
If it works, it works. But consider `build\bootstrap.proj`. Although the benefit of what you've done is it only restores on the CI, in the job that uses it, not on everyone's machine, and every CI job.

Alternatively, there appears to be a supported way to define and restore dotnet repo-local tools: https://docs.microsoft.com/en-us/dotnet/core/tools/local-tools-how-to-use#restore-a-local-tool-installed-by-others